index.d.ts

```diff
@@ -327,8 +327,8 @@ declare module "leap-core" {
   };
 
   class ExtendedWeb3 extends Web3 {
-    public getUnspent(address: string, color: number, cb?: Callback<Array<Unspent>>): Promise<Array<Unspent>>;
-    public getUnspent(address: string, cb?: Callback<Array<Unspent>>): Promise<Array<Unspent>>;
+    public getUnspent(address: string | undefined, color: number, cb?: Callback<Array<Unspent>>): Promise<Array<Unspent>>;
```

Comment:
What about token address instead of color?

Reply:
`address: string | undefined | null` will be better (you use it with `null` in docs)

Reply:
both are great points!